parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -61,6 +56,7 @@
   public static final int DEFAULT_STATISTICS_TRUNCATE_LENGTH = Integer.MAX_VALUE;
   public static final int DEFAULT_PAGE_ROW_COUNT_LIMIT = 20_000;
   public static final int DEFAULT_MAX_BLOOM_FILTER_BYTES = 1024 * 1024;
+  public static final boolean DEFAULT_BLOOM_FILTER_ENABLED = false;
```

Comment:
Why we set the default to false?

Reply:
@chenjunjiedada 
You can find more details about the topic in these issues: [PARQUET-1787](https://issues.apache.org/jira/browse/PARQUET-1787) and [PARQUET-1784](https://issues.apache.org/jira/browse/PARQUET-1784)

Reply:
Thanks @garawalid.